docs/testing/uninstall-add-in.md

```diff
@@ -0,0 +1,142 @@
+
+## Remove a ghost add-in
+
+To remove a ghost add-in, you need to remove the artifacts that were created when it was last sideloaded, remove it's local registration, and for Outlook add-ins remove it's registration in Exchange.
+
+The process for removing the add-in varies depending on whether the add-in is for Outlook or some other Office application. 
+
+> [!NOTE]
+> In the [unified manifest for Microsoft 365](../develop/unified-manifest-overview.md), an add-in can be configured to support Outlook and one or more other Office applications; that is, there is more than one member of the "extensions.requirements.scopes" array in the manifest and one of the members is "mail" (or the "extensions.requirements.scopes" property isn't present). Treat an add-in that is configured in this way as an Outlook add-in.
+
+If the ghost add-in is not an Outlook add-in, skip to the section [Remove the add-in artifacts](#remove-the-add-in-artifacts).
+
+### Remove the Exchange registration of a ghost Outlook add-in
+
+1. Be sure you are logged into Outlook with the same ID you used when you sideloaded the add-in.
+1. Open PowerShell as an Administrator.
+1. Run the following commands. Answer "Yes" to all confirmation prompts.
+
+   ```powershell
+   Install-Module -Name ExchangeOnlineManagement -RequiredVersion 3.4.0
+   Set-ExecutionPolicy RemoteSigned
+   Connect-ExchangeOnline
+   ```
+
+      > [!NOTE]
+      > If the `Connect-ExchangeOnline` command returns the error "ActiveX control '8856f961-340a-11d0-a96b-00c04fd705a2' cannot be instantiated because the current thread is not in a single-threaded apartment", just run the command a second time. This is a well-known bug.
+
+1. Run the following command. Answer "Yes" to all confirmation prompts.
+
+   ```powershell
+   Get-App | Format-Table -Auto DisplayName,AppId
+   ```
+
+   A list of the add-ins installed on Outlook displays. Most of them are built-in Microsoft add-ins, but if you have installed add-ins from other companies, they are listed too as are any ghost Outlook add-ins. 
+
+1.  Find your ghost add-in in the list. If you created it with Yo Office or another Microsoft tool, it probably has the name "Contoso Task Pane Add-in". 
+1. Copy the App ID (a GUID) of the add-in. You need it for later steps.
+1. Run the command `Remove-App -Identity {{The GUID OF YOUR ADD-IN HERE}}`; for example, `Remove-App -Identity 26ead0cb-10dd-4ba2-86c6-4db111876652`. This command removes the add-in from Exchange.
+
+   > [!WARNING]
+   > The removal of the registration needs to propagate to all Exchange servers. Wait at least three hours before continuing with the next step. 
+
+1. Continue with the section [Remove the add-in artifacts](#remove-the-add-in-artifacts).
+
+### Remove the add-in artifacts
+
+> [!IMPORTANT]
+> Carry out this procedure on all devices on which you ever had the add-in sideloaded.
```

Comment:
Something about the tense of this sentence seemed off to me, but I'm not sure this suggestion is better.
```suggestion
> Carry out this procedure on all devices on which you have had the add-in sideloaded.
```